src/nvim/eval/userfunc.c

```diff
@@ -2173,7 +2173,7 @@ void ex_function(exarg_T *eap)
    */
   if (KeyTyped) {
     // Check if the function already exists, don't let the user type the
-    // whole function before telling him it doesn't work!  For a script we
+    // whole function before telling them it doesn't work!  For a script we
```

Comment:
that one's fine (but should be checked whether it's Neovim-specific).

Reply:
Vim has the same text, aka they also use "him"

Reply:
What do you mean by checking if it's neovim-specific? Aka I shouldn't make this change since Vim also uses "him"?